appservice/src/main/java/nu/yona/server/analysis/rest/ActivityControllerBase.java

```diff
@@ -149,7 +149,7 @@
 		try (CryptoSession cryptoSession = CryptoSession.start(password, () -> userService.canAccessPrivateData(userId)))
 		{
 			UserDto user = userService.getPrivateValidatedUser(userId);
-			return messageController.createOkResponse(pagedResourcesAssembler, user, messageSupplier.get());
+			return messageController.createOkResponse(user, messageSupplier.get(), pagedResourcesAssembler);
```

Comment:
Wouldn't it be more natural to have the message as first parameter?

Reply:
Not to me. It's a feeling, I think, and I feel the user should go first and then the messages for that user.